modules/client/src/addresslistVoting/internal/client/methods.ts

```diff
@@ -307,16 +307,18 @@ export class AddresslistVotingClientMethods extends ClientCore
     }
 
     try {
-      await this.graphql.ensureOnline();
+      this.graphql.assertClient();
       const client = this.graphql.getClient();
       const response = await client.request(QueryAddresslistVotingMembers, {
-        address: pluginAddress,
+        address: pluginAddress.toLowerCase(),
       });
       return response.addresslistVotingPlugin.members.map((
         member: { address: string },
       ) => member.address);
-    } catch {
-      throw new GraphQLError("AddressList members");
+    } catch (e) {
+      handleGraphQLError(e as Error, "adddresslistVoting members");
+      await this.graphql.ensureOnline();
+      return this.getMembers(pluginAddress);
```

Comment:
Careful because this can create an infinite recursion loop and also leak memory :warning:

Reply:
Isn't is possible to move the logic into the `request()` primitive and try new clients in loop until one works or until we have tried them all?